code/game/area/almayer.dm

```diff
@@ -691,12 +691,18 @@
 	icon_state = "selfdestruct"
 	fake_zlevel = 1 // upperdeck
 
+/area/almayer/ert_port
+	name = "\improper ERT Docking Port"
+	icon = 'icons/turf/area_almayer.dmi'
```

Comment:
this is already inherited

Reply:
Sure, I just copied this from lifeboat, so I'll make the change there to